packages/@react-aria/focus/src/FocusRing.tsx

```diff
@@ -25,29 +25,15 @@ interface FocusRingProps {
 }
 
 export function FocusRing(props: FocusRingProps) {
-  let {children, focusClass, focusRingClass, within} = props;
```

Comment:
Extracted this logic into a `useFocusRing` hook, which can be used in places where classnames aren't being used for styling.